src/LibraryManager.Vsix/UI/Controls/MutualPropertyChange.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.ComponentModel;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI.Controls
+{
+    internal class MutualPropertyChange : INotifyPropertyChanged
```

Comment:
>INotifyPropertyChanged [](start = 42, length = 22)

Use BindableBase instead.

Reply:
Will do

---
In reply to: [200684735](https://github.com/aspnet/LibraryManager/pull/285#discussion_r200684735) [](ancestors = 200684735)